templates/ts/custom-copilot-rag-custom-api/env/.env.testtool.user.tpl

```diff
@@ -23,5 +23,5 @@ SECRET_AZURE_OPENAI_ENDPOINT='{{{azureOpenAIEndpoint}}}'
 {{^azureOpenAIEndpoint}}
 SECRET_AZURE_OPENAI_ENDPOINT=' '
 {{/azureOpenAIEndpoint}}
-AZURE_OPENAI_DEPLOYMENT=' '
+SECRET_AZURE_OPENAI_DEPLOYMENT=' '
```

Comment:
Why deployment is secret?

Reply:
To align with endpoint. I will confirm with Junjie to see whether we need to keep it as secret.

Reply:
Aligned with current templates. We will save all azure openai parameters in .env.user and keep endpoint and deployment not secret.